common/utils/utils.go

```diff
@@ -225,28 +226,18 @@ func CreateGCSBucket(bucketName, projectID string) error {
 	}
 	defer client.Close()
 	bucket := client.Bucket(bucketName)
-	buckets := client.Buckets(ctx, projectID)
-	for {
-		if bucketName == "" {
-			return fmt.Errorf("bucketName entered is empty %v", bucketName)
-		}
-		attrs, err := buckets.Next()
-		// Assume bucket not found if at Iterator end and create
-		if err == iterator.Done {
-			// Create bucket
-			if err := bucket.Create(ctx, projectID, nil); err != nil {
+	if err := bucket.Create(ctx, projectID, nil); err != nil {
+		if e, ok := err.(*googleapi.Error); ok {
```

Comment:
If the error is non 409, it seems like we are proceeding without returning an error or even logging

Reply:
We were returning errors other than 409 but added a check to return error in case it was not a googleapi error

Reply:
I got it backwards. If the error code is 409 we're not logging anything and returning nil. Is that expected ?

Reply:
If error code is 409 we are printing that existing bucket will be used and that's expected